modules/installation-configuration-parameters.adoc

```diff
@@ -208,28 +191,31 @@ endif::[]
 ifeval::["{context}" == "installation-config-parameters-bare-metal"]
 :bare:
 endif::[]
+ifeval::["{context}" == "installation-config-parameters-ibm-cloud-vpc"]
+:ibm-cloud:
+endif::[]
```

Comment:
Worth noting that while the original implementation of this module also assigned the `ibm-cloud-vpc` attribute, I deliberately removed it. When I implemented that work, I do so for a single sentence related to deploying a cluster to an existing VPC. Removing it was easily accommodated here. [1]

[1]  https://github.com/openshift/openshift-docs/pull/63813/files#diff-14ebe576efb84b2f58b32a0de2996fced237204ae9b091e3cafbd40b65c1ef31R513